subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -349,12 +349,12 @@ static int do_deregister_reply_cb(const struct coap_packet *response,
 	u8_t code;
 
 	code = coap_header_get_code(response);
-	LOG_DBG("Deregister callback (code:%u.%u)",
+	LOG_INF("Deregister callback (code:%u.%u)",
```

Comment:
IMHO these should be LOG_DBG() as typically info level prints just too much data.